src/EditorFeatures/Test/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -51,35 +51,67 @@ protected static bool CanUseSpeculativeSemanticModel(Document document, int posi
             return !service.GetMemberBodySpanForSpeculativeBinding(node).IsEmpty;
         }
 
+        internal static CompletionList GetCompletionList(CompletionListProvider provider, Document document, int position, CompletionTriggerInfo triggerInfo)
+        {
+            var itemsBuilder = ImmutableArray.CreateBuilder<CompletionItem>();
+            Action<CompletionItem> addCompletionItem = item =>
+            {
+                itemsBuilder.Add(item);
+            };
+
+            CompletionItem builder = null;
+            Action<CompletionItem> registerBuilder = item =>
+            {
+                builder = item;
```

Comment:
Do we want to throw if it's already set?  Or is the concept that the last person wins?